pyiron/atomistics/structure/generator.py

```diff
@@ -112,6 +112,30 @@ def create_surface(
         return None
 
 
+def create_hkl_surface(lattice, hkl, layers, vacuum=1.0, center=False):
+    """
+    Create a surface with an arbitrary surface normal (hkl).
+
+    Args:
+        lattice: Atoms bulk instance or str, e.g. "Fe", from
+            which to build the surface
+        hkl (list): miller indices of surface to be created
+        layers (int): # of atomic layers in the surface
+        vacuum (float): vacuum spacing
+        center (bool): shift all positions to center the surface
+            in the cell
+
+    Returns:
+        pyiron.atomistics.structure.atoms.Atoms instance: Required surface
+    """
+
```

Comment:
We need to add the line `s.publication_add(publication_ase())` because we're still using ASE surface creators.